src/main/webapp/css/loginpage.css

```diff
@@ -0,0 +1,83 @@
+
+
+body {
+  text-align:center;
+  margin: 0;
+  font-family: sans-serif;
+  line-height: 1.6;
+  font-size: 18px;
+  line-height: 1.6;
+  color: #444;
+  background-color: #eeeeee;
+}
+
+.navbar-brand {
+  padding: 0 15px;
+  height: 200px;
+  line-height: 80px;
+}
+
+nav {
+  background-color: lightblue;
+  line-height: 3;
+}
+
+
+nav b {
+  text-align: center;
+  background-color: darkslategrey;
+  line-height: 8;
+  column-width: 100000px;
+  color: white;
+  display: inline-block;
+  font-size: 24px;
+  text-decoration: none;
+  font-family: monospace;
+  font-weight: bolder;
+}
+
+#navTitle {
+  font-size: 36px;
+}
+
+#container {
+  margin-left: auto;
+  margin-right: auto;
+  width: 800px;
+}
+
+h1 {
+  color: #757575;
+}
+
+input {
+  font-size: 18px;
+  border: none;
+  border-bottom: 2px solid fuchsia;
+  background-color: #eeeeee;
+  outline: none;
+}
+
+p{
```

Comment:
There will always be a strike of balance on css specificity. Having all <p> tags to be center aligned may be too board and may be confusing in the future. For example, you might add a footer text with some copyright text on the bottom right of the page, this will be automatically centered as well. Consider adding the alignment logic in the top part (known as "Hero" in UX terms), as well as adding to the form part. Although the code is simpler to have everything on the page to be centered, it becomes muddy when some parts do not need to be centered.